### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by pacyL2K19**
## Project 1 : Youtube clone
In this project, we used only html and css.
We used : 
- Grid for the side bar
- Flexbox for the navigation bar and comment section.
- Github workflow to collaborate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,59 @@
...
+
+This project doesn't manage actions when user click on buttons
+
+## Built With
+
+- HTML
+- CSS
+- CSS grid and Flex
+- No framework
+
```
Comment (arslanbisharat) [inline README.md]: - [x] Please remove this from readme.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,59 @@
-# youtube-clone
-The first project on HTML and CSS path on microvrse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Youtube Clone
+
+> This project is a youtube clone - just the frontend
+> It helps student to get confortable with basics of html and css and good practices
+
+This project doesn't manage actions when user click on buttons
```
Comment (arslanbisharat) [inline README.md]: - [x] Please remove this from readme.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,59 @@
...
+## Built With
+
+- HTML
+- CSS
+- CSS grid and Flex
+- No framework
+
+## Live Demo
+
+N/A
```
Comment (arslanbisharat) [inline README.md]: - [x] Please add live link to the readme.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,59 @@
...
+
+
+## Getting Started
+
+1. Clone the project 
+  `$ git clone https://github.com/pacyL2K19/youtube-clone `
+2. Then run it in your prefered browser with the index.html file
```
Comment (arslanbisharat) [inline README.md]: You can rephrased it as "T**hen open index.html in your browser**"

File: README.md [hunk 5]
```diff
@@ -1,2 +1,59 @@
...
+
+- GitHub: [@ahzia](https://github.com/ahzia)
+- Twitter: [@PacifiqueLinja1](https://twitter.com/ZiaYousofi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ah-ziayosfi)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+Feel free to contribute 
+Feel free to check the [issues page](issues/).
```
Comment (arslanbisharat) [inline README.md]: - [x] Please add proper link here

File: index.css [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+  flex-wrap: nowrap;
+}
+
+.avatar {
+  height: 30px;
+  width: 30px;
+}
+/* ------------------- Logo styles ---------------------*/
+/* .logo{
+    height: 30px;
```
Comment (arslanbisharat) [inline index.css]: - [x] Please remove commented code

File: index.css [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+  display: flex;
+  flex-direction: column;
+  padding-top: 5px;
+  padding-bottom: 5px;
+}
+
+.sub-title {
+  /* display: grid;
+  grid-gap: 2px;
+  grid-template-columns: 1fr 1fr; */
```
Comment (arslanbisharat) [inline index.css]: - [x] Please remove commented code

File: index.css [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+  display: flex;
+  align-items: center;
+}
+
+.details {
+  width: 45%;
+  /* grid-column: 1; */
+  /* grid-row: 1; */
```
Comment (arslanbisharat) [inline index.css]: - [x] Please remove commented code

File: index.html [hunk 9]
```diff
@@ -0,0 +1,233 @@
...
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <!-- I suggest we use the flex displaying to better handle the direction and sizes of boxes -->
+    <nav>
+      <!-- In the navbar, we first have image, the first is the 3 bands and the secon is the logo -->
+      <div class="logo-container">
+        <!-- <img class="logo" src="./img/youtube1.png" alt="logo" /> -->
```
Comment (arslanbisharat) [inline index.html]: - [x] Please remove commented code

File: index.html [hunk 10]
```diff
@@ -0,0 +1,233 @@
...
+        placeholder="Search"
+      /><span><button class="btn-search">Search</button></span>
+      <div class="btn-section">
+        <div style="width: 50px"></div>
+        <img src="./img/videoIcon.png" alt="icons" class="icons" />
+        <img src="./img/menuIcon.png" alt="icons" class="icons" />
+        <img src="./img/notifIcon.png" alt="icons" class="icons" />
+        <img src="./img/icons8-male-user-100.png" alt="avater" class="avatar" />
+        <!-- <button>upload</button>
+                <button>Signin</button> -->
```
Comment (arslanbisharat) [inline index.html]: - [x] Please remove commented code

File: index.html [hunk 11]
```diff
@@ -0,0 +1,233 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <!-- I suggest we use the flex displaying to better handle the direction and sizes of boxes -->
+    <nav>
+      <!-- In the navbar, we first have image, the first is the 3 bands and the secon is the logo -->
+      <div class="logo-container">
+        <!-- <img class="logo" src="./img/youtube1.png" alt="logo" /> -->
+        <img src="./img//youtube1.png" alt="youtube logo" height="20px" id="logo" />
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please Avoid using inline-styling

File: index.html [hunk 12]
```diff
@@ -0,0 +1,233 @@
...
+        <img src="./img/notifIcon.png" alt="icons" class="icons" />
+        <img src="./img/icons8-male-user-100.png" alt="avater" class="avatar" />
+        <!-- <button>upload</button>
+                <button>Signin</button> -->
+      </div>
+    </nav>
+    <div class="pageContent">
+        <div class="content">
+          <div class="VideoContainer">
+            <video width="100%" height="400" controls>
```
Comment (arslanbisharat) [inline index.html]: - [ ] Avoid using inline-styling

File: index.html [hunk 13]
```diff
@@ -0,0 +1,233 @@
...
+              <p><b>What is Github?</b></p>
+              <p>ahmad</p>
+              <p>41k view . 3 years ago</p>
+            </div>
+          </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    </div>
```
Comment (arslanbisharat) [inline index.html:226]: Stray end tag **divs**

File: index.html [hunk 14]
```diff
@@ -0,0 +1,233 @@
...
+      <div class="btn-section">
+        <div style="width: 50px"></div>
+        <img src="./img/videoIcon.png" alt="icons" class="icons" />
+        <img src="./img/menuIcon.png" alt="icons" class="icons" />
+        <img src="./img/notifIcon.png" alt="icons" class="icons" />
+        <img src="./img/icons8-male-user-100.png" alt="avater" class="avatar" />
+        <!-- <button>upload</button>
+                <button>Signin</button> -->
+      </div>
+    </nav>
```
Comment (arslanbisharat) [inline index.html]: - [x] Please enclose this part inside **header tag**

File: index.html (hunk 13 above)
Comment (arslanbisharat) [inline index.html]: - [x] Please enclose this part inside main tag

File: index.css [hunk 15]
```diff
@@ -0,0 +1,204 @@
...
+  width: 30px;
+}
+/* ------------------- Logo styles ---------------------*/
+/* .logo{
+    height: 30px;
+} */
+
+#logo {
+  align-self: center;
+}
```
Comment (arslanbisharat) [inline index.css]: - [x] [optional] Please avoid using ids as selectors

File: index.css [hunk 16]
```diff
@@ -0,0 +1,204 @@
...
+.logo-container {
+  flex: 1;
+  padding: 15px;
+}
+.logo-container span {
+  margin-left: 10px;
+  font-weight: bold;
+  font-size: 22px;
+  align-self: center;
+  align-self: center;
```
Comment (arslanbisharat) [inline index.css]: - [x] Please have a look here. You have used same property twice.

File: index.css [hunk 17]
```diff
@@ -0,0 +1,204 @@
...
+/* main container*/
+
+.pageContent {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  padding-left: 7.5%;
+  padding-right: 7.5%;
+  padding-top: 3%;
+  display: grid;
```
Comment (arslanbisharat) [inline index.css]: - [x] Please have a look here. You have used same property twice.

File: index.css [hunk 18]
```diff
@@ -0,0 +1,204 @@
...
+  grid-gap: 20px;
+  grid-template-columns: 7fr 4fr;
+}
+.content {
+width: 95%;
```
Comment (arslanbisharat) [inline index.css]: - [x] Please fix the indentation.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Request changes 😳

Hello Team, 
First of all Congratulations!!! You have made a life changing decision by joining microverse. You are going to enjoy this environment very much. I wish you very best of luck. 

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄
**Please have a look at the suggestions to get more understanding.**

- [x]  Please make your PR descriptive. 
  - Please add more details to it. Please refer to [link ](https://microverse.pathwright.com/auth/sign-in/?next=/library/fast-track-curriculum/69047/path/step/66243642)from the pathway to learn more.
  - Please remove image from your PR message and add it to readme. 


- [x] [Optional] Please add **img** folder and **index.css** file in a single folder. This will help you to make your repo more professional and clean. 



- [x] Please add a live link to your Readme. [Click here](https://thecompetentdev.com/weeklyjstips/tips/7_serve_files_from_github/) to learn how to use https://raw.githack.com to get an instant live link of your project.

- [x] Please have a look at the suggestion I given through inline-comments below. 

- [x] Please try to make more commits. 

- [x] In your project, hamburger is missing. Please work on it. 

**clone**



- [ ] [Optional] You can make all the items including icons and avatar on navbar , channel names, share, save, like, dislike a button as present on original page, 

- [x] Please resolve conflicts between files

- [x] Please make subscribe a button as present on original page. 
---------------------------------------------------------------------------------

**Please make the changes and request re-review for the given project**
_Stay safe and give me a smile 😇_

### Happy Coding
-------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
In case of any question please reach me out on slack: arslanbisharat**.

File: index.css (hunk 17 above)
Comment (ahzia) [reply index.css]: Properties are not the same, do you think we should assign the padding for left and right sides in one line?

Comment (arslanbisharat) [reply index.css]: I am sorry. I mistakenly added padding. It's display that was used twice

File: index.html (hunk 12 above)
Comment (descholar-ceo) [reply index.html]: This Sugested change is not yet implemented

File: index.html [hunk 19]
```diff
@@ -0,0 +1,235 @@
...
+    </header>
+    <main>
+    <div class="pageContent">
+        <div class="content">
+          <div class="VideoContainer">
+            <video width="100%" height="400" controls>
+              <source src="movie.mp4" type="video/mp4" />
+              <source src="movie.ogg" type="video/ogg" />
```
Comment (descholar-ceo) [inline index.html]: These video sources you used here are not working, please make sure that you use real links to real video from youtube, I would sugest you use the `<iframe/>` tag with the embedded video from the real youtube link

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @pacyL2K19 and @ahzia  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes]
- [x] Some of the [required] requested changes from the previous review, were not implemented as requested :point_right: Avoid inline styling in your HTML file
- [x] The video links you added is not working, I highly recommend you to use `<iframe/>` tag with imbedded real youtube link

- [x] Implement all changes I suggested in inline comments in your codes

# [Optional changes]
- [x] You have deleted your readme file from this branch and you pushed it to the `main branch`, that is not allowed, never do it anymore :point_right: it's violating githubflow, let all of the changes on this feature branch until it approved, then you can merge it later, the following is the screenshot showing the two commits you pushed on the main branch, next time try to be careful,
Screenshot from 2020-10-26 22-11-56
- [x] Your commit messages should be in imperative mode or present tense, you should have used `Design the navbar` instead of `Designing the navbar`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

File: index.html (hunk 19 above)
Comment (pacyL2K19) [reply index.html]: Thank you for your suggestion, all of them are fixed, please take a look.

File: index.html [hunk 20]
```diff
@@ -0,0 +1,233 @@
...
+        <img src="./img//youtube1.png" alt="youtube logo" class="logo" />
+      </div>
+      <input
+        class="search"
+        type="text"
+        name="search"
+        placeholder="Search"
+      /><span><button class="btn-search">Search</button></span>
+      <div class="btn-section">
+        <div style="width: 50px"></div>
```
Comment (jubaan) [inline index.html]: This is an inline style attribute. search for this pattern, `style="... some properties..." `, remove it from the HTML file and use the stylesheet to style the affected elementes.

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @pacyL2K19 and @ahzia. Thanks for submitting you project!

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

**The code still has inline `style` attribute in the HTML file. See inline comments**

- [ ] Some of the [required] requested changes from the previous review, were not implemented as requested point_right Avoid inline styling in your HTML file
> * [x]  The video links you added is not working, I highly recommend you to use `<iframe/>` tag with imbedded real youtube link
> * [x]  Implement all changes I suggested in inline comments in your codes


## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

File: index.html (hunk 20 above)
Comment (pacyL2K19) [reply index.html]: Thank you dear for this feedback, all the issues mentioned are now fixed. Please take a look again

File: index.html [hunk 21]
```diff
@@ -0,0 +1,232 @@
...
+                </div>
+              </div>
+              <div class="subscribe-btn">
+                <img src="./img/subscribe.jpg" alt="Subscribe button" />
+              </div>
+            </div>
+          </div>
+          <div class="iconPart"></div>
+          <div class="description">
+            <!-- <h3>Published on Feb 27, 2014</h3> -->
```
Comment (descholar-ceo) [inline index.html]: Please remove this commented code, before you merge changes to the base branch

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @pacyL2K19 and @ahzia , your project is approved :tada: :tada: :tada: 

You have implemented all of the requested changes, [required], this is very good, your project meet all of the minimum requirements, you can go ahead and merge the changes to the base branch,

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

File: index.html (hunk 21 above)
Comment (pacyL2K19) [reply index.html]: Noted dear, thank you

**Pull request merged**